mkosi

```diff
@@ -2266,7 +2266,8 @@ def find_cache(args):
         return
 
     if os.path.exists("mkosi.cache/"):
-        args.cache_path = "mkosi.cache/" + args.distribution.name + "~" + args.release
+        release = "~" + args.release if args.distribution.name != "arch" else ""
```

Comment:
why check explicitly for "arch" here? it appears much nicer to me to just check whether `args.release is not None` or so? i.e. check for what you actually want to check for instead of some secondary condition...

Reply:
Agreed, building other distros should fail/refuse to build before that if they are missing the release arg.